composer.json

```
@@ -28,16 +34,25 @@
     "php": "^7.1 || ^7.2",
     "guzzlehttp/oauth-subscriber": "^0.3"
   },
+  "require-dev": {
+    "phpstan/phpstan": "^0.11.12",
+    "symplify/easy-coding-standard": "^6.0"
+  },
   "scripts": {
     "test": [
-      "@phpunit"
+      "@phpunit",
+      "@fixcs",
+      "@phpstan"
     ],
     "quicktest": [
       "@unit"
     ],
     "phpunit": "../../bin/phpunit -d memory_limit=2048M --bootstrap ../../vendor/autoload.php --configuration phpunit.xml --fail-on-warning --testsuite=all",
     "unit": "../../bin/phpunit -d memory_limit=2048M --bootstrap ../../vendor/autoload.php --configuration phpunit.xml --fail-on-warning --testsuite=unit",
-    "coverage": "../../bin/phpunit -d memory_limit=2048M --bootstrap ../../vendor/autoload.php --configuration phpunit.xml --fail-on-warning --testsuite=all --coverage-text --coverage-html=Tests/Coverage"
+    "coverage": "../../bin/phpunit -d memory_limit=2048M --bootstrap ../../vendor/autoload.php --configuration phpunit.xml --fail-on-warning --testsuite=all --coverage-text --coverage-html=Tests/Coverage",
+    "phpstan": "vendor/bin/phpstan analyse --autoload-file=../../vendor/autoload.php --level=6 Config Connection DependencyInjection Exception Form Integration Mapping Salesforce Tests",
```

The folders here doesn't make sense for this plugin.
```suggestion
    "phpstan": "vendor/bin/phpstan analyse --autoload-file=../../vendor/autoload.php --level=6 Auth Bundle Command Config Configuration Controller DependencyInjection Entity Event EventListener Exception Facade Form Helper Integration Mapping Migration Migrations Services Sync Tests",
```